sdk/communication/communication-common/review/communication-common.api.md

```diff
@@ -91,6 +94,19 @@ export const isPhoneNumberIdentifier: (identifier: CommunicationIdentifier) => i
 // @public
 export const isUnknownIdentifier: (identifier: CommunicationIdentifier) => identifier is UnknownIdentifier;
 
+// @public
+export interface MicrosoftBotIdentifier {
+    cloud?: "public" | "dod" | "gcch";
+    isGlobal?: boolean;
+    microsoftBotId: string;
```

Comment:
it's not updated and this is the reason for the pipeline failure, fix it and the API view will be automatically created for the new commit if there will not be pipeline failure

Reply:
@AikoBB this is an auto-generated file
_> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/)._
I'm afraid we're not supposed to edit it manually. Have you ever edited it manually?

Reply:
nope, I guess the command `rushx build` should update the file after changes to the convenience layer

Reply:
Thanks! It helped